routes/collection/details/[id].tsx

```diff
@@ -1,16 +1,18 @@
-import { Pagination } from "../../islands/datacontrol/Pagination.tsx";
+import { Pagination } from "../../../islands/datacontrol/Pagination.tsx";
```

Comment:
please use the mapping of $islands on these

Reply:
👍